src/main/java/uk/gov/digital/ho/hocs/queue/migration/MigrationCaseService.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.digital.ho.hocs.queue.migration;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.digital.ho.hocs.application.ClientContext;
+
+@Slf4j
+@Service
+public class MigrationCaseService {
+
+    private final MigrationService migrationService;
+    private final ClientContext clientContext;
+    private final MigrationCaseTypeData migrationCaseTypeData;
+    private final String user;
+    private final String group;
+    private final String team;
+
+    @Autowired
```

Comment:
`Autowired` comes by default when you have 1 constuctor.

Reply:
Fixed